components/Vision.tsx

```diff
@@ -0,0 +1,78 @@
+import { styled } from "../stitches.config";
+import { Wrapper } from "@cabindao/topo";
+import ArrowLink from "./ArrowLink";
+
+const Vision = ({
+  title,
+  description,
+  readMoreLink,
+}: {
+  title: string;
+  description: string;
+  readMoreLink?: string;
+}) => {
+  return (
+    <Container id="about">
+      <Wrapper>
+        <Content>
+          <h2>{title}</h2>
+          <div>
+            <p>{description}</p>
+            {readMoreLink && (
```

Comment:
Made this optional as there's no mirror post for passports that I'm aware of yet